lib/geordi/settings.rb

```diff
@@ -81,6 +74,11 @@ def read_settings
       if File.exists?(local_path)
         local_settings = YAML.safe_load(File.read(local_path))
         check_for_invalid_keys(local_settings, ALLOWED_LOCAL_SETTINGS, local_path)
+
+      if File.exists?(File.expand_path("../../.firefox-version"))
+        Geordi::Interaction.warn 'Unsupported setting ".firefox-version". Please remove the file in /home/user/projects/the-project/.firefox-verison.'
```

Comment:
Can you please replace the path with the path from `File.expand_path("../../.firefox-version")`? So the user knows where she needs to delete the file.